comms/storage/client/client.go

```diff
@@ -0,0 +1,106 @@
+package client
```

Comment:
these are really just util functions for seed commands, and we use the term "client" a lot - including for nats clients - so having this be a "client" inside storage feels a little weird. I think it's better to keep the storage directory minimal for storage logic so I'll defer to you but I think it would be nicer to have this just be part of `seed.go` (nothing here actually changes its struct and ClientList is already in seed.go anyway - why not make ClientList a slice of string endpoints and just have these functions take one of the endpoints?)

Reply:
Heard, we do have a lot of things called client. I can change the name of the module to something like `storageclient` but I'm pretty strongly in favor of having a dedicated set of functions for interacting with the server. It might not become super apparent until we do some more thorough integration tests but a go "storage sdk" or "storage client" or whatever the name needs to be is much better than writing raw http requests in our tests

Reply:
okay after renaming it to `storageclient` it feels redundant. It's already in the `storage` module so idk if it needs the added prefix. We can call it storageclient and when we import it, we can import it as `storageclient` but I think Imma keep the module name as just `client`. What're your thoughts?

Reply:
yeah that's ok not a huge deal